src/CardDeck.js

```diff
@@ -0,0 +1,3 @@
+import createWithBsPrefix from './utils/createWithBsPrefix';
+
+export default createWithBsPrefix('card-deck');
```

Comment:
Lets move any component that's simply a `createWithBsPrefix` one into the Parent component (e.g the Card here). There isn't much of an advantage to splitting for cherry-picking and there isn't an logic in these to makethe Card component confusing to read through

Reply:
So the `Card{Deck, Group, Columns}` wrap a bunch of cards together like how a `ButtonGroup` would for buttons, so I think it makes sense for it to be outside of the `Card` component.